recipes/libtar/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+autoreconf --force --install
+./configure --prefix=$PREFIX --disable-debug --disable-dependency-tracking
+make install
```

Comment:
Is there a `make check` we can run here?

Reply:
The Makefiles don't seem to define a check target.

Reply:
I do have a minimal test in the meta.yaml that checks whether the necessary files were installed.  Can we resolve this conversation?